dds/DCPS/RTPS/Util.cpp

```diff
@@ -0,0 +1,37 @@
+#include "dds/DCPS/RTPS/Util.h"
```

Comment:
```suggestion
#include "Util.h"
```

Reply:
Without the path, isn't it ambiguous whether we're talking about DCPS/Util.h and DCPS/RTPS/Util.h? Maybe these need to be named differently, but I'd rather not leave it down to order of search path.

Reply:
No, `#include "file.h"` always searches the directory of the including file first, then goes on to the directories like that ones passed by `-I`.

Reply:
There is also `#include <file.h>` which skips the local check, and just checks the non-local include paths.